src/ui/MessageContent/index.tsx

```diff
@@ -126,6 +129,7 @@ export default function MessageContent(props: MessageContentProps): ReactElement
     renderEmojiMenu = (props: MessageEmojiMenuProps) => <MessageEmojiMenu {...props} />,
     renderEmojiReactions = (props: EmojiReactionsProps) => <EmojiReactions {...props} />,
     renderMobileMenuOnLongPress = (props: MobileBottomSheetProps) => <MobileMenu {...props} />,
+    renderThreadReplies = (props: ThreadRepliesProps) => <ThreadReplies {...props}/>
```

Comment:
Missing `className` and `ref` from this default. Also, we may want to pass the `threadRepliesRef`? The code uses it to calculate component dimensions.

Reply:
yeah i checked those props and don't need em. `threadRepliesRef` looks like it's used to do custom calculation for rendering the sender profile but we use our own component for that